public/js/app/components/statistics/Chart.jsx

```diff
@@ -37,17 +45,21 @@ function Charts({ chartNames = [], languageIndex = 1, schools = {} }) {
     <Row>
       {chartNames.map(numberName => (
         <Col key={numberName} xs="4" style={{ paddingRight: 0, paddingLeft: 0 }}>
-          <Chart data={getChartData(numberName, schools)} label={labels[numberName]} />
+          <Chart data={getChartData(numberName, schools)} label={labels[numberName]} languageIndex={languageIndex} />
         </Col>
       ))}
     </Row>
   )
 }
 
-function Chart({ data = [], label = '' }) {
+function Chart({ data = [], label = '', languageIndex }) {
+  const { allSchools } = i18n.messages[languageIndex].statisticsLabels
   const styles = {
     font: { fontFamily: 'Open Sans', fontSize: '16px' },
   }
+  const tickLable = allSchools.split(' ').join('\n')
+  const formatLabel = t => (t === 'All schools' ? tickLable : t)
+
```

Comment:
Now `allSchools` transformed by tickLable has to come from `data` in the same way as `ITM`, and other schools come.

Reply:
`ITM `is fetched from` schoolsLib.ORDERED_SCHOOLS `that does not require to translate. But `allSchools` needs to translate here and this is why we need to format 'All schools' to split into two lines. If we avoid this one here then we need to pass language object to school.js to transform `allSchools `to` All Schools`. What are your suggestions on this?

Reply:
now it is refactored